Makefile

```diff
@@ -57,6 +76,32 @@ clean:
 	- rm -r bin
 	- rm -r rpmbuild
 
+_unit_test: $(REPORTS)
+	gotestsum $(GO_UNITTEST_FLAGS) $(TEST) $(GINKGO_UNITTEST_FLAGS) -timeout $(TIMEOUT) || ($(MAKE) _post_unit_test && /bin/false)
```

Comment:
Please document (how to install) this dependency in the README (`go install gotest.tools/gotestsum@latest`).

Reply:
Agree. Just a bit wary of the high turnaround times for rebuilds.